diff --git a/node_modules/any-observable/register.js b/node_modules/any-observable/register.js
index 40b8d86..f0097a9 100644
--- a/node_modules/any-observable/register.js
+++ b/node_modules/any-observable/register.js
@@ -26,6 +26,13 @@ function loadImplementation(implementation) {
 	}
 
 	if (!impl) {
+		// Issue with webpack build and library listr (used by @dappnode/dappnodesdk) using any-observable sub-dependency
+		// https://github.com/SamVerschueren/listr/issues/146
+		// https://github.com/okonet/lint-staged/issues/416
+		return {
+			Observable: global.Observable,
+			implementation: 'global.Observable'
+		}
 		throw new Error('Cannot find any-observable implementation nor' +
 			' global.Observable. You must install polyfill or call' +
 			' require("any-observable/register") with your preferred' +
